Timer.java

```diff
@@ -0,0 +1,26 @@
+import greenfoot.*;  
+
+public class Timer extends Actor{
+    private final int SECOND = 60;
+    private final int MINUTE = SECOND*60;
+    private int minutes;
+    private int seconds;
+    public static int timeElapsed;
+
+    public Timer(int minutes, int seconds, int timeElapsed){ 
+        this.minutes = minutes;
+        this.seconds = seconds;
+        this.timeElapsed = timeElapsed;
+    }
+
+    public void act() {
+        watch();
+    }  
+
+    public void watch(){
+        timeElapsed += 1;
```

Comment:
No deben depender de la velocidad del método act para funciones de tiempo, ya que el usuario puede mover la velocidad libremente. Utilicen LocalDateTime https://mkyong.com/java8/java-8-how-to-format-localdatetime/